doc/data_format.rst

```diff
@@ -722,8 +722,13 @@ Offset     Length  Version  Description
   93+s      2       3-      Maximum reliability (in range 0..10000).
   95+s      2       3-      Daily reliability decrease (in range 0..10000).
   97+s      2       3-      Monthly decrease of the maximum reliability (in range 0..10000).
-  99+s      4       1-      Text of the ride (reference to a TEXT block).
- 103+s                      Total length.
+  99+s      4       4-      Base intensity rating.
+ 103+s      4       4-      Base nausea rating.
+ 107+s      4       4-      Base excitement rating.
+ 111+s      4       4-      Excitement increase per working cycle.
+ 115+s      4       4-      Excitement increase per nearby scenery item.
```

Comment:
It would be useful if you add some sort of unit with these fields. Simplest is probably relating values here to the displayed value for the user.

Is "excitement rating" a different unit than "excitement increase"?

In general, if you name some notion "X" (eg at line 725, X = "intensity rating"), then stick to that name everywhere if you intend to refer to the same notion, and use a different name if you intend to refer to a different (but perhaps related) notion. This convention avoids confusion in how many different notions exist, it also makes it more precise in saying which notion is used exactly.

With "increase" at 728/729, are these unsigned values? If not, "change" would be better, as it expresses the option to use a negative value as well. (Although that would be silly here.)